### ISSUE TITLE
Include tlp?

### ISSUE THREAD

**Issue by peteruithoven** [elementary/seed-metapackages#9]
I'm unsure if this is the right place to ask this, please let me know if it's not. 

I'm curious why TLP (and tlp-rdw) aren't included by default in elementaryOS? It seems like almost all "Things to do after installing Elementary OS" articles include it. Are there big downsides? I know some people experience issues with bluetooth and usb devices, but maybe we can find a very conservative set of defaults?

Installing it is very difficult because it's usually recommended to add a ppa and it's not available in the AppCenter because it's a command line tool. Adding the ppa is difficult because there is a lot that can go wrong with ppa's. I understand why Elementary didn't include software-properties-common. 

Maybe the reason people recommend ppa's is an upstream issue, where the ubuntu ppa's tlp version is too far behind the tlp's ppa (version 0.8-1 vs 1.0-1 seems like quite a jump)?

Maybe we can only install / active tlp when the device is a laptop? Some suggest it should be an option during installation, but I don't think most people will have enough information to make an informed decision.

On work that Fedora is putting in on improving battery life: https://www.omgubuntu.co.uk/2018/02/better-battery-life-on-fedora-linux

Are there other utilities we should consider including to improve battery live, like the [Thermal Daemon](https://01.org/linux-thermal-daemon/documentation/introduction-thermal-daemon)?

I've started this discussion earlier on Reddit: https://www.reddit.com/r/elementaryos/comments/7gi6z7/include_tlp_in_elementaryos_by_default/

**Comment by snowparrot** [comment]
Its a very good question!

**Comment by codygarver** [comment]
RE: thermald, it's already installed

RE: tlp
Many years ago I shipped some zram software that improved performance. Then Ubuntu released a kernel update that broke every single elementary OS install because they don't test their kernels against non-standard configurations. Since then I have implemented a policy to not deviate from upstream configuration unless absolutely necessary. Also the tlp package itself could become outdated or suddenly lose support for elementary OS for some reason.
